### ISSUE TITLE
Support a cats.Parallel to support parallel operations on Rerunnable

### ISSUE THREAD

**Issue by johnynek** [travisbrown/catbird#90]
Currently, I don't see a method to take several Rerunnables and run them in parallel (like `IO.Par` allows in cats-effect).

Even a method on the companion: `def parSequence[F[_]: Traverse, A](as: F[Rerunnable[A]]): Rerunnable[F[A]]` would be very nice to have.

**Comment by felixbr** [comment]
I just want to say that this would be really nice, as we've been bitten a few times by `Future.traverseSequentially` and `Future.collect` (composed with `map`) not behaving the same way.

Having `.traverse` and `parTraverse` would make it more explicit and easier to explain to colleagues imo.

**Comment by travisbrown** [comment]
:+1:, I'd happily review a PR.

**Comment by felixbr** [comment]
jfyi, I have a Parallel implementation for Rerunnable on a local branch. I'll write up a PR once I'm somewhere with decent internet.

**Comment by felixbr** [comment]
I think this was resolved with #161. Feel free to complain if there's still something missing, though :)
